presto-native-execution/presto_cpp/main/operators/BroadcastFactory.cpp

```diff
@@ -96,6 +95,11 @@ void BroadcastFileWriter::noMoreData() {}
 
 // TODO: Add file stats - size, checksum, number of rows.
 RowVectorPtr BroadcastFileWriter::fileStats() {
+  // No rows written.
+  if (numRows_ == 0) {
+    return nullptr;
```

Comment:
It would be nice to update comments for this method to explain that it may return nullptr.

Reply:
updated